test/sband/test_sband.py

```diff
@@ -19,48 +19,47 @@
 
 '''  to run > yarn run:test_sband -I uart -d /dev/ttyUSB0 '''
 
-
 import numpy as np
 
 import sys
 from os import path
 sys.path.append("./test")
 
-from testLib import testLib as test
+from tester import Tester
 
-test = test() #call to initialize local test class
+tester = Tester() #call to initialize local test class
 
 def testAllCommandsToOBC():
-    test.sendAndExpect('obc.communication.s_set_control(0 0)', {'err': 0})
-    test.sendAndExpect('obc.communication.s_get_control',
+    tester.sendAndExpect('ex2.communication.s_set_control(0, 0)', {'err': 0})
```

Comment:
There should probably be a way to pass the satellite as a variable instead. Or just omit the satellite entirely and let tester use self.satellite

Reply:
I am not entirely sure what you mean

Reply:
so you send the string "communication.s_set_control(0,0)" and in the tester it prepends "ex2" or whatever else though self.satellite

Reply:
There are cases where the starting string is not ex2 nor any satellite, like `eps.<command>.<command>` 
https://github.com/AlbertaSat/ex2_ground_station_software/blob/96754291e58dedd11805c852d33721fcc4d824d0/test/test_eps.py#L29-L34

Reply:
Ah right, forgot about that. I still think we should have a way to dynamically decide which satellite the test is targeting, but I think it will need more thought